tests/test_jwud_routes.py

```
@@ -0,0 +1,162 @@
+        "source_path": str(source_path),
+        "compressed_path": str(compressed_path),
+        "tmp_path": tmp_path,
+    }
+
+
+@pytest.fixture
+def mock_jwud_service(monkeypatch):
+    """Mock jwudtool_service so the routes never touch the real binary."""
+    mock_service = Mock()
+
+    def fake_info(path):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `fake_info`

(ANN202)